src/react-magic-script/magic-script-renderer.js

```diff
@@ -36,7 +36,14 @@ const UPDATE_SIGNAL = {};
 //  hostContext: HostContext,
 //  internalInstanceHandle: Object
 function createInstance(type, props, rootContainerInstance, hostContext, internalInstanceHandle) {
-  return mxs._nativeFactory.createElement(type, rootContainerInstance, props);
+  try {
+    var obj =  mxs._nativeFactory.createElement(type, rootContainerInstance, props);
+  } catch (e) {
+    let message = `ERROR Type: ${type}\nproperties: ${JSON.stringify(props)}`;
+    console.error(`${message}\n${e.message}`);
```

Comment:
I think @grozdanov uses logError / logWarning for all kinds of output.

Reply:
I would like to have completely independent (from the factory code) which pops the exception message since `createElement` might fail for the `text` element as well. I will pull this branch and re-work it.